### PR TITLE
F change between control modes

### PR DESCRIPTION
**Pull request by kirstyellis**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)

### CODE REVIEW

File: ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp [hunk 1]
```diff
@@ -37,6 +37,8 @@
 
 #include <string>
 #include <vector>
+#include <hardware_interface/actuator_state_interface.h>
```
Comment (toliver) [inline ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp]: Why is this needed here?
It probably should be inside of `actuator_command_interface.h` instead.

**Review by toliver — CHANGES_REQUESTED** [review]

File: ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp (hunk 1 above)
Comment (kirstyellis) [reply ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp]: hmm, good point, it isn't needed here... removing

File: ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h [hunk 2]
```diff
@@ -0,0 +1,87 @@
+///////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2012, hiDOF INC.
+// Copyright (C) 2013, PAL Robotics S.L.
```
Comment (toliver) [inline ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h]: If you have modified something, you should add Shadow to the copyright

**Review by toliver — CHANGES_REQUESTED** [review]

File: ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (C) 2019 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+* Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+*/
```
Comment (toliver) [inline ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h]: This is a public repository and we have licensed it as BSD, so please copy the copyright and license header from another file e.g.
https://github.com/shadow-robot/ros_ethercat/blob/kinetic-devel/ros_ethercat_model/include/ros_ethercat_model/ros_ethercat.hpp#L7-L38

**Review by toliver — CHANGES_REQUESTED** [review]

File: ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h (hunk 3 above)
Comment (toliver) [reply ros_ethercat_model/include/ros_ethercat_model/actuator_command_interface.h]: No need to add author or creation date

**Review by toliver — APPROVED** [review]
